src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/decommission.go

```diff
@@ -0,0 +1,86 @@
+package brokers
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/redpanda"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newDecommissionBroker(fs afero.Fs) *cobra.Command {
+	var force bool
+	cmd := &cobra.Command{
+		Use:   "decommission [BROKER ID]",
+		Short: "Decommission the given broker",
+		Long: `Decommission the given broker.
+
+Decommissioning a broker removes it from the cluster.
+
+A decommission request is sent to every broker in the cluster, only the cluster
+leader handles the request.
+`,
+		Args: cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			broker, err := strconv.Atoi(args[0])
+			out.MaybeDie(err, "invalid broker %s: %v", args[0], err)
+			if broker < 0 {
+				out.Die("invalid negative broker id %v", broker)
+			}
+
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			cl, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			if !force {
+				brokers, err := cl.Brokers(cmd.Context())
+				out.MaybeDie(err, "unable to get broker list: %v", err)
```

Comment:
Can we add `; to bypass the node version check re-run this with --force` to the end of this error?

Reply:
I didn't add it in the first place since the flag will be hidden so no help text about "what the --force flag does".

Should we let the flag visible?

Reply:
Oh, I see that there is another comment to document the force flag in the help text...

Reply:
Hmm should we reveal it to users? It makes it easier that users happen to decommission a node that's in maintenance mode even if it's running < 23.1, which is dangerous, right?

Reply:
It makes it easier to do the dangerous thing, yes. I like hiding it because of two angles here:

* In the long term, users will not need this flag at all, so it becomes increasingly less relevant to expose and document; eventually we should deprecate this
* By hiding it up front, the default behavior for a user is to run this without the flag. Only if things fail will they be notified to bypass the safety check.